experiments/conductor/cmd/runner/scripts/crd_mapper_fuzzer/02-generate-spec-and-status.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+cd $(dirname "$0")
+SCRIPT_DIR=`pwd`
+
+PROMPT=${SCRIPT_DIR}/01-generate-script.prompt
+
+if [[ -z "${WORKDIR}" ]]; then
+  echo "WORKDIR is required"
+  exit 1
+fi
+
+if [[ -z "${BRANCH_NAME}" ]]; then
+  echo "BRANCH_NAME is required"
+  exit 1
+fi
+
+# if [[ -z "${GCLOUD_COMMAND}" ]]; then
+#   echo "GCLOUD_COMMAND is required"
+#   exit 1
+# fi
+
+if [[ -z "${LOG_DIR}" ]]; then
+  echo "LOG_DIR is required"
+  exit 1
+fi
+
+# if [[ -z "${EXPECTED_PATH}" ]]; then
+#   echo "EXPECTED_PATH is required"
+#   exit 1
+# fi
+
+mkdir -p ${LOG_DIR}
+# cat ${PROMPT} | \
+#     envsubst '$GCLOUD_COMMAND,$EXPECTED_PATH' > ${LOG_DIR}/prompt
+
+cd ${WORKDIR}
+
+REPO_ROOT="$(git rev-parse --show-toplevel)"
+cd ${REPO_ROOT}
+
+git co master
+git co ${BRANCH_NAME}
+
+cd apis/${SERVICE}/${CRD_VERSION}
+
+# TODO: We should be able to in-place replace here
+# TODO: Generate spec and status separately
```

Comment:
FYI, I had a type-generator-v2 which does this. I'm not working on it at the moment, but when I tested it on `ManagedKafkaCluster`, the generated types required no manual edits.

https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/3748